oper8/watch_manager/ansible_watch_manager/resources/roles/oper8_app/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
   # 1) Define the 'develop_shell' ansible variable in the playbook invocation
   # 2) Define the per-playbook 'enable_ansible_vcs' variable
   # 3) Set the ANSIBLE_VCS environment variable
-  when: develop_shell is not defined and "{{ enable_ansible_vcs | default('true', True) }}" != "false" and "{{ lookup('env', 'ANSIBLE_VCS') | default('true', True) }}" != "false"
+  when: develop_shell is not defined and "enable_ansible_vcs | default('true', True)" != "false" and "lookup('env', 'ANSIBLE_VCS') | default('true', True)" != "false"
```

Comment:
Have we checked that these functions are still called inside quotes? Like I'm a little worried that ansible is just doing a direct string comparison between `"enable_ansible_vcs | default('true', True)"` and `"false"`

Reply:
Hm, yeah, we definitely need to check this one too.